notary-api/src/utils/wibson-lib/cryptography/hashing.js

```diff
@@ -73,11 +73,14 @@ const signPayload = (privateKey, ...args) => {
   return signMessage(privateKey, messageHash);
 };
 
+const sha3 = message => web3.utils.sha3(message);
```

Comment:
Why doesn't just destructure sha3 ?

Reply:
I was doing the same as `packMessage`, since web3 is not exported